src/components/varsom-observation/varsom-observation.tsx

```diff
@@ -639,7 +632,7 @@ Observation={obs}
 
 {obs._incident ? 
 <varsom-incident
-strings={this.strings}
+
```

Comment:
Pirk: Ser ut som du har erstattet strings={this.strings} med en blank linje en del steder. Det er kanskje penere hvis du får til å ikke ha en blank linje her?

Reply:
Topp, ser mye ryddigere ut nå.